npm-shrinkwrap.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.10.0",
   "dependencies": {
     "@okta/courage": {
-      "version": "2.1.0"
+      "version": "2.5.0-beta.0"
```

Comment:
Hmm, I'm not sure if we have a rule for this, but why not publish a new version of courage and include a release version in here? It's not that big of a deal now, but that's the ideal process in the future; doesn't seem like too much extra work to standardize on it.